controllers/component_build_controller.go

```diff
@@ -329,6 +391,9 @@ func (r *ComponentBuildReconciler) Reconcile(ctx context.Context, req ctrl.Reque
 
 	switch requestedAction {
 	case BuildRequestTriggerSimpleBuildAnnotationValue:
+		updateMetricsTimes(componentIdForMetrics, requestedAction, reconcileStartTime)
+		initialBuildStatus := readBuildStatus((&component))
```

Comment:
Why pass the component like this `(&component)`? The parentheses should not be required.

Reply:
removed